docs/docs/migration.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Migration
+sidebar_position: 9
+---
+
+## Migrating existing usage from 1.x to 2.0
```

Comment:
Nit: Markdown best practices call for a line break between headings and subsequent content.

https://www.markdownguide.org/basic-syntax/#heading-best-practices